spec/menufactory_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menufactory'
+require 'menu'
+require 'dish'
+
+describe MenuFactory do
+
+  let(:list) { [{name: :sushi, price: 5},
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.